python/mujincontrollerclient/uriutils.py

```diff
@@ -443,6 +443,9 @@ def __init__(self, **kwargs):
         # guess suffix based on primary key
         if not self._suffix and self._primaryKey.endswith(b'.mujin.dae'):
             self._suffix = u'.mujin.dae'
+        elif self._suffix and self._primaryKey.endswith(self._suffix):
+            log.warn('suffix specified in both the primary key and as a parameter')
+            self._primaryKey = self._primaryKey[:-len(self._suffix)]
```

Comment:
This seems not correct. Why do you remove suffix from primary key?  @cv3d

Reply:
@bucketzxm You are right. I fixed that, and I also provided an example to reproduce the bug.